server/controllers/gh.js

```diff
@@ -33,26 +36,24 @@ function createIssue(req, res) {
 
     return got.post(issuesRequestUrl, {
         body: {
-            body: commentData.text,
-            title: commentData.title
+            title: commentData.title,
+            body: commentData.text
         },
-        token: req.user && req.user.accessToken
-    })
-        .then(postData => {
-            res.status(200).send(postData.body);
-        })
+        token: getToken(req.user)
+    }).then(postData => res.status(200).send(postData.body))
         .catch(err => onError(req, res, err));
 }
 
 function getIssues(req, res) {
     logger.log('getIssues');
 
-    const token = req.user && req.user.accessToken;
-
     Promise.all([
-        makeIssueRequest(issuesRequestUrl, { token, query: req.query }),
-        makeLabelsRequest(labelsRequestUrl, { token })
-    ]).then(function(responses) {
+        makeIssueRequest(issuesRequestUrl, {
+            query: req.query,
+            token: getToken(req.user)
+        }),
+        makeLabelsRequest(labelsRequestUrl, { token: getToken(req.user) })
```

Comment:
`getToken(req.user)` is called twice for the same request. it's better to cache it into variable (and the same bellow)